### PR TITLE
documentation(installation): Add Hints about @vue/composition-api

### PR DESCRIPTION
**Pull request by FlorianWendelborn**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,12 +44,16 @@ or
 **Documentation**: [current release](https://3yourmind.github.io/kotti/) or [current master](https://kotti.netlify.com/)
 
 ```typescript
-// in main.js / entrypoint
+// in main.ts / entrypoint
```
Comment (carsoli) [inline README.md:47]: arguably we should just say in your entry point, no specification of a ts must or a js must
and maybe no specification of `main` altho it's ofc the default if we spin up a vue project

or what do u think

File: README.md [hunk 2]
```diff
@@ -44,12 +44,16 @@ or
...
 import Vue from 'vue'
 import KottiUI from '@3yourmind/kotti-ui'
 import '@3yourmind/kotti-ui/dist/styles.css'
 
-// register all KtComponents globally
+// also make sure to set-up @vue/composition-api as it’s required for newer Kotti features
+import VueCompositionAPI from '@vue/composition-api' // right now, 0.5.0 is recommended
+Vue.use(VueCompositionAPI)
```
Comment (carsoli) [inline README.md:54]: should we stress on plugin order as a comment? 
also is it verified to be an issue?

File: README.md (hunk 1 above)
Comment (FlorianWendelborn) [reply README.md:47]: But it says `/ entrypoint`?

File: README.md (hunk 2 above)
Comment (FlorianWendelborn) [reply README.md:54]: What do you mean "verified to be an issue"?

Comment (carsoli) [reply README.md:54]: if we use the composition plugin after using kotti-ui

Comment (carsoli) [reply README.md:54]: does it actually prove to be a problem

Comment (FlorianWendelborn) [reply README.md:54]: Ah, that’s what you mean. It’s at least conceptually a problem and I don’t want to rely on the order not mattering in the global magic land that vue is. I can add a comment for that then I guess.

File: README.md (hunk 1 above)
Comment (carsoli) [reply README.md:47]: yes im retarded :)

File: README.md [hunk 3]
```diff
@@ -44,12 +44,16 @@ or
...
 ```typescript
-// in main.js / entrypoint
+// in main.ts / entrypoint
 import Vue from 'vue'
 import KottiUI from '@3yourmind/kotti-ui'
 import '@3yourmind/kotti-ui/dist/styles.css'
 
-// register all KtComponents globally
+// also make sure to set-up @vue/composition-api as it’s required for newer Kotti features
+import VueCompositionAPI from '@vue/composition-api' // right now, 0.5.0 is recommended
```
Comment (carsoli) [inline README.md:53]: we specify it to be 0.5.x on our peer deps
maybe we shouldn't be that specific and the best we can add is 'pls no use of beta'

Comment (carsoli) [reply README.md:53]: so even higher patches are supported and i think we already have that in yoda if i'm not mistaken

just saying maybe let's not overkill with specificity but also leave room for a beta-fuckup

Comment (carsoli) [reply README.md:53]: not an essential change just a suggestion and the last i have 

so i will approve already

Comment (FlorianWendelborn) [reply README.md:53]: https://github.com/3YOURMIND/kotti/blob/cc115881976692d3d14504ab2430fb1a3a63f8ab/packages/kotti-ui/package.json#L50

**Pull request merged**